### PR TITLE
Update pokemon_v2/README.md

### PR DESCRIPTION
**Pull request by sh0hei**

### CODE REVIEW

File: pokemon_v2/README.md [hunk 1]
```diff
@@ -1285,7 +1285,7 @@ Moves are the skills of pokémon in battle.  In battle, a Pokémon uses one move
...
 | past_values    | A list of move resource value changes across ersion groups of the game                                                                                                    | list [PastMoveStatValues](#pastmovestatvalues) |
 | stat_changes   | A list of stats this moves effects and how much it effects them                                                                                                           | list [MoveStatChange](#movestatchange) |
-| contest_effect | The effect the move has when used in a super contest                                                                                                                      | [NamedAPIResource](#namedapiresource) ([ContestEffect](#contest-effects)) |
+| super_contest_effect | The effect the move has when used in a super contest                                                                                                                      | [NamedAPIResource](#namedapiresource) ([ContestEffect](#contest-effects)) |
```
Comment (SerenadeX) [inline pokemon_v2/README.md:1288]: So there are three different types of contest effects.  
- Contest: Gen III
- Super Contest: Gen IV
- Contest Spectacular: Gen VI, specifically ORAS

They are different based on the generation.  Should we distinguish between them?

Comment (phalt) [inline pokemon_v2/README.md:1288]: @SerenadeX you are definitely right. @zaneadix is an expert on the data we've got here, so he might have some input :)

**Comment by zaneadix** [comment]
@SerenadeX thanks for finding this. Could you also update the link? ([ContestEffect](#contest-effects)) should also point to Super Contest Types

I would take the Contest Spectacular discussion over to https://github.com/veekun/pokedex

I believe they're still updating their data to completely incorporate ORAS. Seems to get more time consuming every time a new game comes out because things just get more complex. Sun and Moon is going to kill us :)

**Comment by phalt** [comment]
YEY I can't wait for Sun and Moon haha.

**Comment by phalt** [comment]
@sh0hei please rebase against master :)

**Comment by sh0hei** [comment]
@phalt done :)

**Comment by sh0hei** [comment]
Thanks :)

**Pull request merged**